src/app/pages/OpenWalletPage/Features/FromLedger/__tests__/index.test.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from 'react'
+import { render, act, screen } from '@testing-library/react'
+import { FromLedger } from '..'
+import { MemoryRouter } from 'react-router-dom'
+
+jest.mock('react-redux', () => ({
+  useSelector: jest.fn(),
+  useDispatch: jest.fn(),
+}))
+
+jest.mock('react-router-dom', () => ({
+  ...jest.requireActual('react-router-dom'),
+  useNavigate: () => jest.fn(),
```

Comment:
Why you are assigning a mock function here?

Reply:
Honestly don't remember anymore - this also goes for some other changes. Will need to look into it in more detail, what were the decisions behind the reasoning.